pysrc/pip_resolve.py

```diff
@@ -246,7 +247,7 @@ def is_testable(requirement):
 
 def detect_encoding_by_bom(path):
     with open(path, 'rb') as f:
-        raw = f.read(4)  # will read less if the file is smaller
+        raw = f.read(NUMBER_OF_BYTES)  # will read less if the file is smaller
```

Comment:
With this one I was more referring to WHY are you reading 4 bytes 😄 Maybe call it `BOM_BYTES` or something like that? Naming is hard.